HerPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -5,10 +5,199 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-xl">
-            Privacy notice
+            ‘Check if you are eligible for an energy grant for a home with no gas boiler (Home Upgrade Grant)’ Privacy Notice
         </h1>
         <p class="govuk-body">
-            TODO: Fill in this page
+            This notice is provided by the Department for Energy Security and Net Zero and sets out how we will use your personal data, and
+            your rights. It is made under Articles 13 and/or 14 of the UK General Data Protection Regulation (UK GDPR). 
         </p>
+        <p class="govuk-body">
+            This service is for citizens to find out whether they might be eligible for the HUG2 grant and, if they are, to refer them to 
+            their local authority. 
+        </p>
+        <p class="govuk-body">
+            This privacy notice only relates to how the Department for Energy Security and Net Zero (DESNZ) uses your data in connection with
+            the service and covers information saved as you proceed through the application and after you submit your referral. 
+        </p>
+        <p class="govuk-body">
+            DESNZ and local authorities are the independent data controllers for your data. This privacy notice does not cover local authorities
```

Comment:
I thought they weren't allowed to shorten it to DESNZ?

Reply:
So did I :)